### PR TITLE
FirstAlert ZCOMBO Smoke Alarm

### PR DESCRIPTION
**Pull request by parasthakkar-newell**
Integration of 500 series

### CODE REVIEW

File: devicetypes/smartthings/zwave-smoke-alarm.src/zwave-smoke-alarm.groovy [hunk 1]
```diff
@@ -21,7 +21,7 @@ metadata {
 
 		attribute "alarmState", "string"
 
-		fingerprint mfr:"0138", prod:"0001", model:"0002", deviceJoinName: "First Alert Smoke Detector and Carbon Monoxide Alarm (ZCOMBO)"
+		fingerprint mfr:"0138", prod:"0001", model:"0003", deviceJoinName: "First Alert Smoke Detector and Carbon Monoxide Alarm (ZCOMBO)"
```
Comment (tpmanley) [inline devicetypes/smartthings/zwave-smoke-alarm.src/zwave-smoke-alarm.groovy:25]: Thanks for the PR. Can you leave the existing fingerprint and just add the new one?

**Comment by parasthakkar-newell** [comment]
I need to add new one.
Leave the existing one as it is.

**Comment by tpmanley** [comment]
@parasthakkar-newell do you have the new version of this device? I don't see it on the Z-Wave certified product list

**Comment by parasthakkar-newell** [comment]
Tom,

Certification is in progress at Pepper-one lab.

Thanks,
Paras

**Comment by parasthakkar-newell** [comment]
Got these changes merged in another pull request https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/12335, so closing this one

**Pull request closed without merging**